Client/src/Remote/Endpoints/DatasetEndpoint.tsx

```diff
@@ -49,6 +49,25 @@ export const approvedDataset = async (query: IFlaggedDatasetQuery) => {
     SnackbarUtils.success(`Dataset ${query.datasetId} was approved!`)
   }
 }
+
+export const callGetUserFavouriteDatasets = async (): Promise<number[]> => {
+  return get(userSavedDatasetsRoute).json()
```

Comment:
this should be awaited, i dont understand why sonar told you that it shouldnt .json() returns a Promise, and Promises are always awaited

Reply:
I thought the same thing but sonar insisted, I will return it back now